build.psm1

```diff
@@ -1091,6 +1091,71 @@ function Start-PSBootstrap {
     }
 }
 
+function Start-PSRelease {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string]$Name, # i.e. v6.0.0-alpha.12
+        [Parameter(Mandatory)]
+        [string]$CommitSHA1 # i.e. 7985366f9ec43af47b24dd88f04c0afc5eab3825
+    )
+
+    Push-Location $PSScriptRoot
+    try {
+        # check that we are in the right location
+        if ($IsWindows) {
+            if ($PSScriptRoot -ne "C:\PowerShell") {
```

Comment:
How necessary is this? We can create packages out of arbitrary directories can't we?

If it is required, what about this instead?
(Join-Path $env:systemdrive "PowerShell")